smart-mode-line.el

```diff
@@ -515,6 +515,13 @@ If this variable is nil, nothing is displayed."
   :group 'smart-mode-line-others
   :package-version '(smart-mode-line . "1.16"))
 
+(defcustom sml/not-modified-char (char-to-string
+                                  (if (char-displayable-p ?✓) ?✓ ? ))
```

Comment:
Unfortunately, this character is not fixed-width. For instance, in my font (monaco) it's a little wider than a regular character, so the modeline moves around when the user saves/edits the buffer.

Just leave the default as `" "`